_docs/collaboration.md

```diff
@@ -5,23 +5,24 @@ title: Collaboration
 order: 1
```

Comment:
I think, when paging through docs in order, Collaboration would logically follow Account Setup and all the other general docs pages.

Current headings:
![flow](https://user-images.githubusercontent.com/29528527/74776972-76f5c880-524d-11ea-9019-120fc8b82196.png)


```suggestion
order: 6
```